src/Yoti/YotiClient.php

```diff
@@ -179,31 +177,32 @@ public function performAmlCheck(AmlProfile $amlProfile)
 
     /**
      * Make REST request to Connect API.
+     * This method allows to stub the request call in test mode.
```

Comment:
To mock out the Request Object it has to be injected in the constructor of the YotiClient::class which is not the case here.
Don't get me wrong this can be achieve by modifying the constructor to receive the request object as a parameter, but that will make it hard for people who integrate Yoti to use this client as they have one more step added to their use case.